test/readStreamTest.js

```diff
@@ -0,0 +1,93 @@
+var tape = require('tape');
+var event_stream = require('event-stream');
+var fs = require('fs-extra');
+
+var readStream = require('../src/readStream');
+
+tape('readStream', function(test) {
+  /*
+    this test is not terribly attractive, i'm not happy with it but setup wasn't
+    all that painful.
+  */
+  test.test('readStream should return from all requested types and populate wofRecords', function(t) {
+    // remove tmp directory if for some reason it's been hanging around from a previous run
+    fs.removeSync('tmp');
```

Comment:
I would pull all the file writing and dir creating into a setup function so it's more clear where the test actually begins.